docker-compose.yml

```diff
@@ -1,18 +1,8 @@
-version: '3'
+# This docker-compose file is made for development purpose and build an image from source, if you want to use Invidious in production, see the documentation.
```

Comment:
```suggestion
# Warning: This docker-compose file is made for development purposes.
# Using it will build an image from the locally cloned repository.
#
# If you want to use Invidious in production, see the [docker-compose.yml file provided
# in the installation documentation: https://docs.invidious.io/Installation.md
```